narwhals/translate.py

```diff
@@ -403,18 +403,6 @@ def _from_native_impl(  # noqa: PLR0915
     allow_series: bool | None = None,
     version: Version,
 ) -> Any:
-    from narwhals._arrow.dataframe import ArrowDataFrame
-    from narwhals._arrow.series import ArrowSeries
-    from narwhals._dask.dataframe import DaskLazyFrame
-    from narwhals._duckdb.dataframe import DuckDBInterchangeFrame
-    from narwhals._ibis.dataframe import IbisInterchangeFrame
-    from narwhals._interchange.dataframe import InterchangeFrame
-    from narwhals._pandas_like.dataframe import PandasLikeDataFrame
-    from narwhals._pandas_like.series import PandasLikeSeries
-    from narwhals._polars.dataframe import PolarsDataFrame
-    from narwhals._polars.dataframe import PolarsLazyFrame
-    from narwhals._polars.series import PolarsSeries
-    from narwhals._spark_like.dataframe import SparkLikeLazyFrame
```

Comment:
moving these deeper should reduce the chances of unnecessary imports happening by accident

yes, it's already tested, but i'm paranoid